website/blog/2019-05-28-hola.md

```diff
@@ -0,0 +1,11 @@
+---
+id: hola
+title: Hola
+author: Gao Wei
+author_title: Docusaurus Core Team
+author_url: https://github.com/wgao19
+author_image_url: https://avatars1.githubusercontent.com/u/2055384?v=4
+tags: [hola, docusaurus]
+---
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque elementum dignissim ultricies. Fusce rhoncus ipsum tempor eros aliquam consequat. Lorem ipsum dolor sit amet
```

Comment:
Should we clean up all the generated files ?

Reply:
I left them to use it as a reference. We should definitely delete them after we added the basic pages.